scripts/scalafmt

```diff
@@ -3,7 +3,7 @@
 set -e
 
 HERE="`dirname $0`"
-VERSION=$(sed -nre "s#version[^0-9]+([0-9.]+)#\1#p" $HERE/../.scalafmt.conf)
+VERSION=$(sed -nre "s#[[:space:]]*version[^0-9]+([0-9.]+)#\1#p" $HERE/../.scalafmt.conf)
```

Comment:
Sorry to repeat myself.  Has this been run manually on Windows?
We, you & I, are going to win no friends if we break SN development on
Windows.

CI runs on Ubuntu Linux. Unless we know that this has been successfully manually run on Windows, we should assume it broken.

As mentioned, I can run this on Windows if such would be hard for you.

Reply:
I'm not sure what's the build env on windows. I've tried on mingw64 and it worked, I don't know what exact `sed` implementation uses SN on Windows. To be 100% sure it's better if you try yourself.

Reply:
Now that I look at this more closely, I realize that I am a goat, the barnyard critter, not the 
"Greatest of all time" kind.

This is obviously a bash script, which will not run on plain Windows to begin with. Arrgh!  :facepalm:        
It might be run on WSL (linux on Windows), but any `sed` there should handle the POSIX
`[[:space:]]`

Sorry for the false trigger.